test/Npgsql.Tests/Replication/SafeReplicationTestBase.cs

```diff
@@ -104,7 +108,9 @@ async Task SafeReplicationTestCore(Func<string, string, string, Task> testAction
                     await DropSlot();
                 }
 
-                await c.ExecuteNonQueryAsync($"DROP PUBLICATION IF EXISTS {publicationName}");
+                if (c.PostgreSqlVersion > Pg10Version)
```

Comment:
Should this be greater or equals? We also have some extension methods over Version in TestUtil in case that looks better.

Reply:
> Should this be greater or equals?

Yeah, you're right. It should be `>=`.

> We also have some extension methods over Version in TestUtil in case that looks better.

Thanks for the hint. I'll use them.

Reply:
> Thanks for the hint. I'll use them.

Nope. I can't use them because they issue `Assert.Ignore()` which isn't what we want here.

Reply:
I was referring to [this] - it's just an extension over Version without Assert.Ignore (it would make the Pg10Version variable unnecessary). But up to you, it's not important.

Reply:
> I was referring to [this]

Did you mean to put a link in there?
I'm still not sure what you mean.